src/app/address/address.page.html

```diff
@@ -95,7 +97,7 @@
               [(ngModel)]="form.streetName"
               required
               name="streetName"
-              placeholder="Your street name"
+              placeholder="Street name only (e.g Squires Lane)"
```

Comment:
it's missing the text on top?

Reply:
Do we need text as well if the placeholder is being instructive?